src/session/session_api.c

```diff
@@ -1735,6 +1735,12 @@ __session_verify(WT_SESSION *wt_session, const char *uri, const char *config)
     SESSION_API_CALL(session, ret, verify, config, cfg);
     WT_ERR(__wt_inmem_unsupported_op(session, NULL));
 
+    /* FIXME-WT-14553: Implement verify for disagg. */
+    if (__wt_conn_is_disagg(session)) {
+        __wt_verbose_warning(session, WT_VERB_VERIFY, "%s", "skipped verify due to disagg");
```

Comment:
Do we really need to print a warning in this case? If customers are going to call verify for disagg, I think they can expect that normal case shouldn't cause extra output. I thought the intention now is to pretend that we do the verification, but it returns OK for all the tables (even for corrupted ones) :)

WDYT about downgrading the log level to some level that is not printed by default?

```suggestion
        __wt_verbose_info(session, WT_VERB_VERIFY, "%s", "skipped verify due to disagg");
```

Reply:
I agree that `warning` could be too much. However, I checked what was done for the other features that are disabled in disag for now, and copied the same pattern. Here are examples I found in the code:
```
        __wt_verbose_warning(session, WT_VERB_RTS, "%s", "skipped recovery RTS due to disagg");
                    __wt_verbose_warning(session, WT_VERB_RTS, "%s", "skipped shutdown RTS due to disagg");
```

Reply:
Had a quick chat with @korteland. At the time they proceeded with using `warning` level because skipping RTS was considered a correctness risk. For verify, Will suggests dropping to an `info` level is more appropriate. 
```suggestion
        __wt_verbose_info(session, WT_VERB_VERIFY, "%s", "skipped verify due to disagg");
```